### DISCUSSION TITLE
How to create a justified grid inside a 'q-page'

### DISCUSSION THREAD

**Discussion by Stopfield** [quasarframework/quasar#16057] [General - Components / Directives / etc] [answered]
Hi, sorry to bother yall!
I'm trying to create a `q-page` with a grid layout, similar to this:

| ┌───┐ | ┌───┐ |
| │░░░│ | │░░░│ |
| └───┘ | │░░░│ |
| ┌───┐ | │░░░│ |
| │░░░│ | │░░░│ |
| └───┘ | └───┘ |
| ┌───┐ | ┌───┐ |
| │░░░│ | │░░░│ |
| └───┘ | └───┘ |

I'm trying to insert `q-card`s with Vega-Lite charts for a visualization project (those boxes above).
In the left side, I try to use `column` with `justify-evenly` so the cards are aligned vertically, but it doesn't work at all.
How do I proceed? I'm completely lost. Also, how do I create a box bigger in proportion to the other ones (regarding the right side)?

**Comment by dongwa** [comment, marked as answer]
Do you want this?
![image](https://github.com/quasarframework/quasar/assets/38457491/5d7f43af-9fac-466a-af2a-78120d9622bc)


Code:
```vue
<template>
  <q-page class="flex q-gutter-md">
    <div class="col flex column justify-evenly q-gutter-md">
      <div class="bg-red col">1</div>
      <div class="bg-red col">2</div>
      <div class="bg-red col">3</div>
    </div>
    <div class="col column justify-evenly q-gutter-md">
      <div class="bg-red col-10">4</div>
      <div class="bg-red col">5</div>
    </div>
  </q-page>
</template>
```
